internal/backends/postgresql/collection.go

```diff
@@ -314,6 +314,8 @@ func (c *collection) Explain(ctx context.Context, params *backends.ExplainParams
 
 	res.QueryPushdown = where != ""
 
+	q += where
```

Comment:
If sort pushdown was enabled, explain didn't work before :sweat_smile:

Reply:
Nice catch! Note for myself: our tests for `EnableSortPushdown` **work** :)